subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/CacheLayout.java

```diff
@@ -72,7 +72,7 @@ public enum CacheLayout {
         .changedTo(96, "6.4-rc-1")
         .changedTo(97, "6.8-rc-1")
         .changedTo(99, "7.5-rc-1")
-        .changedTo(100, "8.0-rc-1")
+        .changedTo(100, "8.0-milestone-5")
```

Comment:
An unrelated question, why the number doesn't increment continuously? `82` -> `95` -> ... `97` -> `99`?